### ISSUE TITLE
Sales Form accepts invalid input.

### ISSUE THREAD

**Issue by Srkahlon** [RasaHQ/rasa-demo#334]
Hi,

I started with the sales form and it asked me to add, "job function".
I added an email address instead and it took it as job function and also as email slot.
Using enter_data to accept all form values, might not come as the intent with the highest confidence always, and the slots cannot accept anything and everything.

Image for reference -

![image](https://user-images.githubusercontent.com/20045478/64236743-acfdf500-cf18-11e9-8246-6fc3d5a6c399.png)

**Comment by akelad** [comment]
Thanks for reporting that - yeah that's not quite desired behaviour. Right now the job function just accepts any sort of input

**Comment by Srkahlon** [comment]
The Slot Mapping for it makes it take any input when the intent is enter_data, and likewise, email is detected as enter_data intent.
Also, email gets added in the email id slot aswell.

**Comment by akelad** [comment]
yeah i know, that's the way it is now - we may work on changing that in future. But not right now

**Comment by melindaloubser1** [comment]
@akelad this is still relevant; do we want to plan for it?

**Comment by akelad** [comment]
how many `contact_sales` intents have there been recently?

**Comment by melindaloubser1** [comment]
107 in the last month

**Comment by akelad** [comment]
then probably yeah :P put it in the inbox. Wouldn't say it's high prio though
